src/models/vtr/classifier.py

```diff
@@ -61,7 +62,7 @@ def __init__(
         )
         self.encoder = nn.TransformerEncoder(encoder_layer, num_layers=num_layers)
         self.norm = nn.LayerNorm(hidden_size)
-        self.classifier = nn.Linear(hidden_size, 1)
+        self.classifier = nn.Linear(hidden_size, num_classes)
```

Comment:
As I understand, you mean this line: https://github.com/kategimranova/vitrina/blob/0e23354fa6258a3da70fde54137bb1331d1af130/src/models/vtr/classifier.py#L78
Now number of classes is always >= 2, so it works properly, because `squeeze(1)` is just doing nothing